code/modules/reagents/chemistry_properties/prop_positive.dm

```diff
@@ -574,19 +574,32 @@
 /datum/chem_property/positive/defibrillating/process_dead(mob/living/M, potency = 1, delta_time)
 	if(!ishuman(M))
 		return
-	var/mob/living/carbon/human/H = M
-	H.apply_damage(-H.getOxyLoss(), OXY)
-	if(H.check_tod() && H.is_revivable() && H.health > HEALTH_THRESHOLD_DEAD)
-		to_chat(H, SPAN_NOTICE("You feel your heart struggling as you suddenly feel a spark, making it desperately try to continue pumping."))
-		playsound_client(H.client, 'sound/effects/Heart Beat Short.ogg', 35)
-		addtimer(CALLBACK(H, TYPE_PROC_REF(/mob/living/carbon/human, handle_revive)), 50, TIMER_UNIQUE)
-	else if (potency > POTENCY_MAX_TIER_1 && H.check_tod() && H.is_revivable() && H.health < HEALTH_THRESHOLD_DEAD) //Will heal if level is 7 or greater
-		to_chat(H, SPAN_NOTICE("You feel a faint spark in your chest."))
-		H.apply_damage(-potency * POTENCY_MULTIPLIER_LOW, BRUTE)
-		H.apply_damage(-potency * POTENCY_MULTIPLIER_LOW, BURN)
-		H.apply_damage(-potency * POTENCY_MULTIPLIER_LOW, TOX)
-		H.apply_damage(-potency * POTENCY_MULTIPLIER_LOW, CLONE)
-		H.apply_damage(-H.getOxyLoss(), OXY)
+	var/mob/living/carbon/human/dead = M
+	if(dead.check_tod() && dead.is_revivable() && dead.health > HEALTH_THRESHOLD_DEAD )
```

Comment:
might want to cache the result of `dead.check_tod() && dead.is_revivable()` since it's used on 578 and 582

Reply:
assuming by "caching" you mean creating two vars and using that instead